WalletWasabi.Fluent/Models/PrivacyLevel.cs

```diff
@@ -0,0 +1,8 @@
+namespace WalletWasabi.Fluent.Models;
+
+public enum PrivacyLevel
+{
+	Private = 1,
+	SemiPrivate = 2,
+	NonPrivate = 3,
+}
```

Comment:
@SuperJMN in which case can `Invalid` happen?

Reply:
Just a nitpick. It's useful to detect bugs. When the enum is not initialized it will default to 0, and making the compiler show "Invalid" instead of 0 is a good idea :)